SpatialGDKEditorToolbar/Source/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -79,13 +76,81 @@ bool CheckClassNameListValidity(const TArray<UClass*>& Classes)
 }
 }// ::
 
-void GenerateSchemaFromClasses(const TArray<UClass*>& Classes, const FString& CombinedSchemaPath)
+bool CheckVariableNameValidity(TSharedPtr<FUnrealType> TypeInfo)
+{
+	// Check Replicated Data
+	FUnrealFlatRepData RepData = GetFlatRepData(TypeInfo);
+
+	for (EReplicatedPropertyGroup Group : GetAllReplicatedPropertyGroups())
```

Comment:
I'd rather not allow any variable name collision, even if they're in different replicated groups, or one is replicated while the other is handover, just for consistency.
The reason is that it may be otherwise confusing why e.g. variables `Jumped?` and `Jumped!` don't clash if one of them is `OwnerOnly` but do clash if both or neither of them are.